### PR TITLE
Stop live services requesting to go live

### PR DESCRIPTION
**Pull request by klssmith**
Live services shouldn't be able to request to go live again. Once a service is live we remove the option to go live from the Settings page, but we still link to the page to request to go live from other places e.g. the 'Get started' page. As a result, we've seen some services make a request to go live when their service has already been live for months - this change will stop that from happening.

@karlchillmaid is going to look at the content for this

_Screenshot now updated with the agreed content_

Screenshot 2020-11-06 at 17 54 11

### CODE REVIEW

File: app/templates/views/service-settings/service-already-live.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+{% endblock %}
+
+{% block maincolumn_content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      {{ page_header('Your service is already live') }}
+
+      <p class="govuk-body">
+        {{ current_service.name }} is already live.
+      </p>
```
Comment (karlchillmaid) [inline app/templates/views/service-settings/service-already-live.html:15]: The h1 and first line of body content are very similar. We should avoid repetition if we can.

**Solution 1** – make the service name part of the h1

```suggestion
      {{ page_header('‘{{ current_service.name }}’ is already live') }}
```

Issue: we know that some service names can be very long

**Option 2** – remove the service name altogether

```suggestion
      {{ page_header('Your service is already live') }}
```

Issue: while this looks neater, it’s less helpful – especially if the user doesn’t notice the current service name in the breadcrumb.

Comment (karlchillmaid) [reply app/templates/views/service-settings/service-already-live.html:15]: We discussed another option, which was to make the first line of the body more useful. For example:

```‘Service name’ went live on Thursday 5 November 2020.```

We didn‘t go with this because it would make the code more complicated for a page which is very rarely seen (@klssmith can only remember seeing this happen twice).

Comment (klssmith) [reply app/templates/views/service-settings/service-already-live.html:15]: Actually, I've just taken another look and it is a very easy change to add the date and doesn't complicate things after all.

Comment (karlchillmaid) [reply app/templates/views/service-settings/service-already-live.html:15]: Great, let’s go with:

```suggestion
      {{ page_header('Your service is already live') }}

      <p class="govuk-body">
        ‘{{ current_service.name }}’ went live on [day] [date] [month] [year].
      </p>
```

File: app/templates/views/service-settings/service-already-live.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+{% block maincolumn_content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      {{ page_header('Your service is already live') }}
+
+      <p class="govuk-body">
+        {% if current_service.go_live_at %}
+          ‘{{ current_service.name }}’ went live on {{ current_service.go_live_at | format_date_normal }}.
+        {% else %}
+          ‘{{ current_service.name }}’ is already live.
```
Comment (klssmith) [inline app/templates/views/service-settings/service-already-live.html:17]: There are a few services in the database which are live (i.e. `restricted` is `False`) but that don't have a `go_live_at` timestamp. These are almost all services created by people in Notify, so it is very unlikely that anyone will end up in this `else` clause and see a message without the date.

File: app/templates/views/service-settings/service-already-live.html (hunk 1 above)
Comment (klssmith) [reply app/templates/views/service-settings/service-already-live.html]: Ok, that's updated now - I've updated the screenshot to show the new message.

**Review by karlchillmaid — APPROVED** [review]
Content looks good – thank you!

File: app/templates/views/service-settings/service-already-live.html (hunk 2 above)
Comment (CrystalPea) [reply app/templates/views/service-settings/service-already-live.html:17]: ok, should we still test that this edge case renders ok? (as in there is no error)

Comment (klssmith) [reply app/templates/views/service-settings/service-already-live.html:17]: I tried to cover that in `test_request_to_go_live_redirects_if_service_already_live` - it's parameterized to test the page when a service does have `go_live_at` and also when `go_live_at` is `None`

Comment (CrystalPea) [reply app/templates/views/service-settings/service-already-live.html:17]: Perfect, I see it now, sorry!

**Review by CrystalPea — APPROVED** [review]
